openwisp_users/tests/test_admin.py

```diff
@@ -90,9 +90,9 @@ def test_admin_change_user_email_empty(self):
         })
         response = self.client.post(reverse('admin:openwisp_users_user_change', args=[admin.pk]), params)
         queryset = User.objects.filter(username='testchange')
-        self.assertEqual(queryset.count(), 0)
+        self.assertEqual(queryset.count(), 1)
         self.assertEqual(len(mail.outbox), 0)
-        self.assertContains(response, 'errors field-email')
+        self.assertEqual(response.status_code, 302)
```

Comment:
this is not correct, we still want the email field to be required.
Users cannot create users without the email field. Only the system itself can (eg: the django-freeradius module).